replacerfunc: read notes before merging

apawlisch opened the pull request:
We had an error when converting multiple choice questions to json and this adds a function into our survey service that handles that. We haven't been able to test this completely since all of our environments either time out or don't start up properly, however we don't get the same error as before. Someone with a working environment please test this before merging with main.

KyleBricker committed:
multiple choices showing up

Commit:
doing stuff
Merge branch 'multipleChoice' of https://github.com/AutoSurvey/AutoSurvey-Frontend

Commit:
merging with main
Merge branch 'main' of https://github.com/AutoSurvey/AutoSurvey-Frontend

KyleBricker committed:
choicecomponent formcontrol stuff hooked up

Commit:
merging
Merge branch 'multipleChoice' of https://github.com/AutoSurvey/AutoSurvey-Frontend

KyleBricker committed:
submitted survey included multiple choices

Commit:
merging
Merge branch 'multipleChoice' of https://github.com/AutoSurvey/AutoSurvey-Frontend

Commit:
try 2

mcinerneym reviewed (changes requested):
The multiple choices are not saving in the survey

mcinerneym committed:
Found a solution to the survey choices problem

Commit:
merge conflicts

Commit:
trying to fix merge conflicts
Merge branch 'timeoutTest' of https://github.com/AutoSurvey/AutoSurvey-Frontend into timeoutTest

mcinerneym reviewed (approved):
Works good.
